internal/driver/nodeserver.go

```diff
@@ -241,28 +242,28 @@ func (ns *NodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	defer done()
 
 	volumeID := req.GetVolumeId()
-	stagingTargetPath := req.GetStagingTargetPath()
-	log.V(2).Info("Processing request", "volumeID", volumeID, "stagingTargetPath", stagingTargetPath)
+	log.V(2).Info("Processing request", "volumeID", volumeID)
```

Comment:
question: Any reason for removing stagingTargetPath here?

Reply:
added it back